benchmarks/data-serving/client/Dockerfile

```diff
@@ -1,27 +1,20 @@
-FROM cloudsuite/java:openjdk8
-
-ENV CASSANDRA_VERSION 3.11.3
+FROM cloudsuite/cassandra:4.0
 
 ENV YCSB_VERSION 0.14.0
 
 RUN apt-get update \
- && apt-get install gnupg software-properties-common wget -y \
- && buildDeps="python  cassandra-tools=$CASSANDRA_VERSION" \
- && set -x \
- && apt-key adv --keyserver ha.pool.sks-keyservers.net --recv-keys A278B781FE4B2BDA \
- || apt-key adv --keyserver ipv4.pool.sks-keyservers.net --recv-keys A278B781FE4B2BDA \
- && echo 'deb http://www.apache.org/dist/cassandra/debian 311x main' >> /etc/apt/sources.list.d/cassandra.list \
- && apt-get update \
- && apt-get install -y $buildDeps --no-install-recommends \
- && apt-get autoremove && apt-get clean && apt-get upgrade -y
+    && wget -q --show-progress --progress=bar:force -O /tmp/cassandra-tools_${CASSANDRA_VERSION}_all.deb https://dl.bintray.com/apache/cassandra/pool/main/c/cassandra/cassandra-tools_${CASSANDRA_VERSION}_all.deb \
```

Comment:
Are these necessary? I mean this image is built on top of the Cassandra image. That image doesn't have the tools that are being installed in the following lines?

Reply:
Client uses cqlsh which is part of cassandra-tools package. As this is only used in client, it is not included in base image

Reply:
I see, OK! 
Does it actually need something that is in the Cassandra image? If not, we should just use Java as the base image.

Reply:
Updated

Reply:
We had this discussion before and the base image was changed to Java as it didn't need anything from the Cassandra image, but it's now back to Cassandra. Does it really need to be Cassandra or it only needs Java?

Reply:
cassandra-tools is requiring cassandra to be installed. If the base image is java, the tools are not getting installed properly. hence changed back to cassandra base image.

Reply:
In the previous version, we didn't have Cassandra installed though for the client. 🤔 
Are you sure this is required?

Reply:
That is because previously we were installing from apt-get which automatically installs cassandra. Now we are installing from dpkg, so we needed cassandra installed already hence the cassandra base image.